pyproject.toml

```
@@ -24,6 +24,7 @@ python-dotenv = "^1.0.0"
 inquirer = "^3.1.4"
 openpyxl = "^3.1.2"
 logging = "^0.4.9.6"
+streamlit = "^1.31.0"
 
```

We also need matplotlib:

```suggestion
streamlit = "^1.31.0"
matplotlib = "^3.8.0"

```